earthpy/tests/test_spatial.py

```diff
@@ -162,12 +163,81 @@ def test_bytescale_high_low_val():
 def test_stack_invalid_out_paths_raise_errors():
     """ If users provide an output path that doesn't exist, raise error. """
     with pytest.raises(ValueError, match="not exist"):
-        es.stack_raster_tifs(
+        es.stack(
             band_paths=["fname1.tif", "fname2.tif"],
             out_path="nonexistent_directory/output.tif",
         )
 
 
+def test_stack_raster(basic_image_tif):
+    """Unit tests for raster stacking with es.stack()."""
+
+    # create list of 4 basic_image_tif files (filepaths)
+    test_files = [basic_image_tif] * 4
```

Comment:
Since `test_files` and `band_files` are identical, do we need both @joemcglinchy? Sorry if I am missing something.

Reply:
ahhh good catch @mbjoseph i'm seeing the same thing.